scriptworker/constants.py

```diff
@@ -179,6 +179,7 @@
         "gecko-2-decision",
         "gecko-3-decision",
         "gecko-decision",  # legacy
+        "gecko-focus",  # mozilla-mobile releases
```

Comment:
This seems like a misnomer - Focus is not Gecko, correct?
It doesn't block anything, but we may want to spin up a 2nd workerType at some point with a more accurate name, and then retire gecko-focus.